src/apps/vpn/telemetry.cpp

```diff
@@ -127,6 +130,20 @@ void Telemetry::connectionStabilityEvent() {
        {"transport", vpn->networkWatcher()->getCurrentTransport()}});
 }
 
+void Telemetry::startTimeToMainScreenTimer() {
```

Comment:
This does not need to be static. You can call Telemetry::instance()->startTime...